### PR TITLE
Just a slight change to the divider style.

### PR DESCRIPTION
**Pull request by kellegous**
screen shot 2015-01-31 at 12 48 41 pm

### CODE REVIEW

**Comment by kellegous** [comment]
@michalsanger What do you think of this divider?

**Comment by michalsanger** [comment]
@kellegous looks good to me

File: pub/assets/js/hound.js [hunk 1]
```diff
@@ -639,7 +639,9 @@ var FilesView = React.createClass({
             </a>
           </div>
           <div className="file-body">
-            {matches}
+            <div className="matches">
```
Comment (michalsanger) [inline pub/assets/js/hound.js]: Maybe you can discard these changes since you don't use ```.matches``` in your css

**Comment by kellegous** [comment]
Good idea to remove .matches. I also fixed an issue where the border was a bit busted.
screen shot 2015-01-31 at 3 45 31 pm

@jklein any objections?

**Comment by jklein** [comment]
I like the idea, although in your most recent screenshot the divider is extremely subtle. I almost can't see it on my monitor. Can we darken it a little bit?

**Comment by kellegous** [comment]
Cool. Made it a little darker.

**Comment by michalsanger** [comment]
Ups, the ```matches``` div had purpose, see #50

**Pull request merged**